lib/integrations/base.rb

```diff
@@ -16,6 +16,14 @@ def self.key
       raise 'key must be defined in the child class'
     end
 
+    def self.required_settings
+      @required_settings ||= Integration.find(key).fetch('settings').map do |setting|
```

Comment:
It would raise an error before it gets to this point:
https://github.com/rainforestapp/rainforest-integrations/blob/develop/app/models/integration.rb#L12